product_docs/docs/efm/4.1/efm_user/07_using_efm_utility.mdx

```diff
@@ -188,16 +188,15 @@ Invoke the `efm node-status-json` command to display the status of a local node
 
 The following is an example output of the `efm node-status-json` command:
 
-> ```text
-> {
->   "type":"Standby",
->   "address":"172.16.144.130",
->   "agent":"UP",
->   "db":"UP",
->   "vip":"",
->   "vip_active":"false"
-> }
-> ```
+ ```text
```

Comment:
Weird that this shows all lines changed instead of the one removed. Am guessing it's a whitespace change but as long as it is ok in the generated docs I don't care if it's *exactly* the same as before or not.